app/views/bookmarks/_filters.html.erb

```diff
@@ -128,6 +128,11 @@
         </dl>
       </dd>
       <dt class="landmark"><%= ts("Submit") %></dt>
+      <dd class="submit actions"><%= link_to ts("Clear Filters"), original_path(@collection) if @collection %></dd>
+      <dd class="submit actions"><%= link_to ts("Clear Filters"), original_path(@tag) if @tag %></dd>
+      <dd class="submit actions"><%= link_to ts("Clear Filters"), original_path(@fandom) if @fandom %></dd>
+      <dd class="submit actions"><%= link_to ts("Clear Filters"), original_path(@pseud) if @pseud %></dd>
+      <dd class="submit actions"><%= link_to ts("Clear Filters"), original_path(@user) if @user and not @pseud %></dd>
```

Comment:
Let's deemphasize the "Clear Filters" links a bit by moving it out of the `<dl>` and making it into a small plain text link. Putting `<p class="footnote"><a href="URL">Clear Filters</a></p>` under the `</dl>` should work. 

You'll probably also need to reposition the logic regarding which link to display -- I noticed the current approach was leaving a lot of empty `<dd>` tags in the code, and we'd rather not have empty elements lying around.